sdk/monitor/monitor-opentelemetry-exporter/src/config.ts

```diff
@@ -27,6 +27,11 @@ export interface AzureExporterConfig {
    * Azure Active Directory Credential
    */
   aadTokenCredential?: TokenCredential;
+
+  /**
+   * Determines if we are exporting StatsBeat
+   */
+  isStatsbeat?: boolean;
```

Comment:
This is a user exposed configuration, is there a scenario where they may want this flag?

Reply:
Good point, I'll just pass this boolean as a separate parameter to the AzureMonitorMetricExporter's constructor.